backend/src/packages/meditation/meditation.controller.ts

```diff
@@ -117,23 +117,20 @@ class MeditationController extends BaseController {
    *                  message:
    *                    $ref: '#/components/schemas/MeditationEntryResponse'
    *        400:
-   *          description: Bad request (Invalid format)
+   *          description: Bad request or Payload too large
```

Comment:
Not sure about Payload too large

Reply:
What do you mean? That's one of the response that we get
![adding_id_1](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/f92d007d-abef-4238-8eb7-f810f1c51ade)